api/DOMMatrix.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "DOMMatrix": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrix",
+        "support": {
+          "webview_android": {
+            "version_added": "61"
+          },
+          "chrome": {
+            "version_added": "61"
+          },
+          "chrome_android": {
+            "version_added": "61"
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "33"
+          },
+          "firefox_android": {
+            "version_added": "33"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "48"
+          },
+          "opera_android": {
+            "version_added": "48"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
```

Comment:
It looks like the "Available in Web workers" (typically as the `worker_support` feature) didn't get migrated from the table. Would you mind adding it in here?